packages/agreement-process/test/utils.ts

```diff
@@ -83,6 +120,10 @@ export const addOneTenant = async (tenant: Tenant): Promise<void> => {
   await writeInReadmodel(tenant, tenants);
 };
 
+export const addOneAttribute = async (attribute: Attribute): Promise<void> => {
+  await writeAttributeInEventstore(attribute);
```

Comment:
No need to write the attribute in the event store, since here we are testing only Agreement events - in addOneAttribute you can just write in readmodel as we do for Tenant / EService in this test suite

Reply:
copy/paste leftover, cleaned 🧹